services/webhooks/service.go

```diff
@@ -43,22 +43,24 @@ func (s *service) Call(session flows.Session, request *http.Request) (*flows.Web
 		}
 	}
 
+	// If user has explicitly set Accept-Encoding: gzip, remove it as Transport will add this itself,
+	// and it only does automatic decompression if its the one to set it.
+	if request.Header.Get("Accept-Encoding") == "gzip" {
```

Comment:
A user was running into this problem yesterday - they were setting `Accept-Encoding` explicitly - usually `Transport` does this automatically and then transparently decodes the response.. if you set it explicitly.. there's no automatic decompression.. and we get a binary response which eventually blows up when mailroom tries to save it as UTF8 in the database.

Reply:
Oh man, subtle!